### PR TITLE
Add DSLS license server support

### PR DESCRIPTION
**Pull request by julianaklulo**
#### What
Provide support to the new `DSLS` license server.

#### Why
`RLM` will be replaced by `DSLS`.

`Task`:  https://jira.scania.com/browse/ASP-5000

---

#### Peer Review
Please follow the upstream omnivector documentation concerning
[peer-review guidelines](https://github.com/omnivector-solutions/Documentation/blob/main/Contributing/pr_review_standards.md#peer-review).

### CODE REVIEW

**Comment by matheushent** [comment]
Do you mind adding a bit of context in the commit message? Our [reference](https://doc.clickup.com/18022949/p/h/h60h5-6891/1fc40157827997d) for commit messages

File: src/license_manager_agent_ops.py [hunk 1]
```diff
@@ -207,7 +207,7 @@ def _upgrade_license_manager_agent(self, version: str):
             self._PIP_CMD,
             "install",
             "--upgrade",
-            f"self._PACKAGE_NAME=={version}",
+            f"{self._PACKAGE_NAME}=={version}",
```
[the remark this answers is not in the document]
Comment (julianaklulo) [reply src/license_manager_agent_ops.py]: Opened #67

**Comment by julianaklulo** [comment]
> Do you mind adding a bit of context in the commit message? Our [reference](https://doc.clickup.com/18022949/p/h/h60h5-6891/1fc40157827997d) for commit messages

@matheushent is the commit message better now?

File: src/license_manager_agent_ops.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (matheushent) [reply src/license_manager_agent_ops.py]: perfect, it's approved

**Comment by matheushent** [comment]
> > Do you mind adding a bit of context in the commit message? Our [reference](https://doc.clickup.com/18022949/p/h/h60h5-6891/1fc40157827997d) for commit messages
> 
> @matheushent is the commit message better now?

Perfect, thank you!

**Review by matheushent — APPROVED** [review]

**Pull request merged**